src/sentry/integrations/slack/sdk_client.py

```diff
@@ -119,9 +120,13 @@ def __init__(self, integration_id: int):
             # this is a read operation.
             """
             with in_test_hide_transaction_boundary():
```

Comment:
oh this is gross. We may want to revisit this transaction boundary in the future 😬